### PR TITLE
Create pow.py

### LINKED ISSUE #126: [tutorial] Add pow tutorial
**Description of the tutorial**
add pow function tutorial 

**Do you want to be assigned to work on this?**
(mark [x])
- [ ] YES
- [x] NO

**Optional: Any additional info you want to add**
...

### PR DESCRIPTION
**Pull request by HenokB**
### 2. Description
(Write description of your changes here)

### 3. Fill in checklist by marking [x]

- [ ] I've read the CONTRIBUTING.md
- [x] I was assigned to this issue
- [ ] My code is formatted using required linting
- [ ] I've run my code locally and checked it works

### CODE REVIEW

File: common-built-in-functions/pow.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# ------------------------------------------------------------------------------------
+# Tutorial: pow() built-in function in Python
+# ------------------------------------------------------------------------------------
+
+# Code here explaining concept with comments to guide
+
+
+#Return base to the power exp; if mod is present, return base to the power exp, modulo mod (computed more efficiently than pow(base, exp) % mod). 
+#The two-argument form pow(base, exp) is equivalent to using the power operator: base**exp.
```
Comment (sabderemane) [inline common-built-in-functions/pow.py:9]:
```suggestion
# Pow is a function which takes 2 arguments (numbers) : base and exp. There is also an optional parameter called mod.
# It return base to the power exp; if the parameter mod is present, it return base to the power exp, modulo mod (computed more efficiently than pow(base, exp) % mod). 
# The two-argument form pow(base, exp) is equivalent to using the power operator: base**exp.
```

**Review by sabderemane — CHANGES_REQUESTED** [review]
Thank you @HenokB for your contribution 😄 There are few suggestions suggested to improve the tutorial. There are also some errors from the linter you can see [here](https://github.com/inspirezonetech/TeachMePythonLikeIm5/pull/125/checks). Don't hesite to read the [CONTRIBUTING](https://github.com/inspirezonetech/TeachMePythonLikeIm5/blob/main/CONTRIBUTING.md#use-flake8-linting-to-ensure-format-of-code-is-consistent-with-repo) guide to fix those issues.

**Comment by HenokB** [comment]
Thanks and I've applied all your suggestions.

**Review by sabderemane — COMMENTED** [review]
It's missing only the linter errors and after that it will be okay

**Comment by HenokB** [comment]
> linter
what kind of linter?

**Comment by sabderemane** [comment]
The flake8 linter, one of the check failing
You can see [here](https://github.com/inspirezonetech/TeachMePythonLikeIm5/pull/125/checks). Don't hesitate to read the [CONTRIBUTING](https://github.com/inspirezonetech/TeachMePythonLikeIm5/blob/main/CONTRIBUTING.md#use-flake8-linting-to-ensure-format-of-code-is-consistent-with-repo) guide to fix those issues.

**Comment by sabderemane** [comment]
@HenokB any news on that ?

**Comment by funbeedev** [comment]
@sabderemane I'll add the hacktoberfest-accepted label for now.
@HenokB We'd appreciate if you made the requested changes soon :)

**Pull request closed without merging**